packages/progress-circle/src/ProgressCircle.ts

```diff
@@ -101,6 +114,12 @@ export class ProgressCircle extends SizedMixin(SpectrumElement, {
         `;
     }
 
+    private onSlotChange(): void {
```

Comment:
Think of a way to make this a helper method. This is repeated many number of times.

Reply:
This is a great question! It may be a good idea to go this PR and then follow up with a learning spike to look at how not just this, but a good amount of functionality shared between these three components could be abstracted for simpler maintenance.